stl/inc/generator

```diff
@@ -311,14 +311,13 @@ private:
             _STL_INTERNAL_STATIC_ASSERT(is_pointer_interconvertible_base_of_v<_Base, _CoroPromise>);
 #endif // ^^^ no workaround ^^^
 
-            _Base& _Current = _Handle.promise();
-            if (!_Current._Info) {
+            _Nest_info* _Info = _Handle.promise()._Try_get_nest_info();
+            if (!_Info) {
                 return _STD noop_coroutine();
             }

```

Comment:
Style suggestion: This is a minimal transformation from the existing code (a good thing, considered in isolation), but I think a slightly larger change would be even better. The other uses are scoping `_Info` to an `if`-statement, and that would seem to be natural here too.